participant-manager/README.md

```diff
@@ -31,6 +31,9 @@ To deploy the [`Participant manager`](/participant-manager/) manually:
 1. [Create](https://cloud.google.com/compute/docs/instances/create-start-instance) a Compute Engine VM instance with your preferred machine type and OS (for example, e2-medium and Debian 10), then [reserve a static IP](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-internal-ip-address)
 1. Verify that your VM instance has the `Stackdriver Logging API` write [access scope](https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam) (on by default) and that your VM’s [service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) has the [`Logs Writer`](https://cloud.google.com/logging/docs/access-control) role (off by default)
 1. Check out the latest code from the [FDA MyStudies repository](https://github.com/GoogleCloudPlatform/fda-mystudies/)
+1. Update the landing page logo as desired replacing the image [`landing-logo.png`](src/assets/images/branding/landing-logo.png) (ensure you keep the same file name)
```

Comment:
Make these 3 new lines sub-bullets of a line that says 1. Optional. Customize branding by updating images and text

Reply:
Updated.

Reply:
Is this updated? I'm not seeing the sub-bullets?

Reply:
Hi @jkrogers18  sub-bullets are updated.
![image](https://user-images.githubusercontent.com/59555917/108195407-ba736300-713d-11eb-85e5-93e8ce142738.png)